scripts/prow/README.adoc

```diff
@@ -1,110 +1,147 @@
+----
+$ brew link --force gettext
+----
 
-. Switch to you `openshift-docs` directory.
+.Procedure
 
+. Switch to your `openshift-docs` directory.
 . Check out the `main` branch, fetch from upstream, and rebase against upstream.
-
-. Create the new branch config by running the `add-new-prow-config.sh` script passing the `$VERSION`, `$BRANCH` and `$DISTROS` variables, for example:
+. To create the new branch config, run the `add-new-prow-config.sh` script passing the `$VERSION`, `$BRANCH` and `$DISTROS` variables, for example:
 +
 [source,terminal]
 ----
 ./scripts/prow/add-new-prow-config.sh 4.15 enterprise-4.15 "openshift-enterprise openshift-rosa openshift-rosa-hcp openshift-dedicated microshift"
 ----
 +
-[NOTE]
-====
-Quote multiple distros to include them as a single item in the build config.
-====
-+
 The script copies the new job YAML to the ci-operator config folder in the `$HOME/release` directory.
 +
 [IMPORTANT]
 ====
-`openshift-rosa`, `openshift-rosa-hcp`, and `openshift-osd` distros should be included in main, current, and future enterprise branch builds only.
+`openshift-rosa`, `openshift-rosa-hcp`, and `openshift-osd` distros should be included in `main`, current, and future enterprise branch builds only.
 When you add a new enterprise branch, remember to remove `openshift-rosa`, `openshift-rosa-hcp`, and `openshift-osd` distros from the most recent enterprise branch.
 
 For example, when the current version is 4.15, and you create a enterprise-4.16 branch, remove `openshift-rosa` and `openshift-osd` distros from the enterprise-4.15 branch build.
-====
-+
-[NOTE]
-====
-You can optionally add an `openshift-docs-vale-review` and `openshift-docs-jira-links` test step to the `validate-asciidoc` job.
-For more information, see link:https://steps.ci.openshift.org/job?org=openshift&repo=openshift-docs&branch=main&test=validate-asciidoc[openshift-docs test steps].
+
+For more information about how to add an `openshift-docs-vale-review` and `openshift-docs-jira-links` test step to the `validate-asciidoc` job, see link:https://steps.ci.openshift.org/job?org=openshift&repo=openshift-docs&branch=main&test=validate-asciidoc[openshift-docs test steps].
 ====
 
-. Change to the `$HOME/release` directory and verify that the new build config has been added.
+. Switch to the `$HOME/release` directory.
+. To verify that the new build config has been added, run the following command by using the prow job you created in the previous step, for example:
 +
-[source,text]
+[source,terminal]
 ----
-$HOME/release/ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-enterprise-4.15.yaml
+$ ls /home/agantony/release/ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-rhacs-docs-4.8.yaml
 ----
 
-. Run the following commands from the root of the `$HOME/release` repository to generate the rest of the Prow build configuration:
+. To generate the rest of the Prow build configuration, run the following commands:
```

Comment:
I think maybe this should be backed out to match the previous, with all the commands in one block.